modules/blogging/src/Volo.Blogging.Web/Pages/Blogs/Posts/Edit.cshtml.cs

```diff
@@ -56,7 +56,8 @@ public async Task<ActionResult> OnPostAsync()
                 Url = Post.Url,
                 CoverImage = Post.CoverImage,
                 Content = Post.Content,
-                Tags = Post.Tags
+                Tags = Post.Tags,
+                Description = !Post.Description.IsNullOrWhiteSpace() ? Post.Description : Post.Content.Substring(0,200)
```

Comment:
`ArgumentOutOfRangeException` is thrown when the content field length is less than 200